scripts/bootstrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
I would go for a `Makefile` WDYT?

Reply:
I don't think this could be reasonable to add since we don't have to download it as dependency (we are using a docker image that doesn't have it by default) and for one step, we can just use the bash script.

If you still think it's better, tell me and I'll do it :)